addons/mail/static/src/components/attachment_image/attachment_image.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+import { useComponentToModel } from '@mail/component_hooks/use_component_to_model/use_component_to_model';
+const { Component } = owl;
+
+export class AttachmentImage extends Component {
+
+    setup() {
+        super.setup();
+        useComponentToModel({ fieldName: 'component', modelName: 'mail.attachment_image', propNameAsRecordLocalId: 'attachmentImageLocalId' });
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {mail.attachment_image}
+     */
+    get attachmentImage() {
+        return this.messaging && this.messaging.models['mail.attachment_image'].get(this.props.attachmentImageLocalId);
+    }
+
+}
+
+Object.assign(AttachmentImage, {
+    defaultProps: {
+        imageSize: '200x200',
+        isEditable: true,
+    },
+    props: {
+        attachmentImageLocalId: String,
+        /**
+         * Image size in server format (eg. 200x200)
+         */
+        imageSize: {
+            type: String,
+            validate: imageSize => {
+                return imageSize.split('x').every(n => Number.isInteger(Number(n)));
+            }
```

Comment:
Just nitpicking, but `200x200x200` passes (shouldn't) and `200x200.5` doesn't (should).
But doesn't really matter: motivation for validate is to prevent typos, and it does the job well enough.

Haven't checked what's gonna happen by just passing `200`. It's considered valid but it's invalid for the route `/web/image`...
Honestly it's overkill to ensure all cases, so let's keep it simple.

Reply:
Need to add a condition that the length result of the split is 2 to prevent most of these.